src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -534,65 +341,20 @@ static UCS_CLASS_INIT_FUNC(uct_cuda_copy_iface_t, uct_md_h md, uct_worker_h work
         return status;
     }
 
-    self->id                     = ucs_generate_uuid((uintptr_t)self);
-    self->config.max_poll        = config->max_poll;
-    self->config.max_cuda_events = config->max_cuda_events;
-    self->config.bw              = config->bw;
+    self->id                       = ucs_generate_uuid((uintptr_t)self);
+    self->config.bw                = config->bw;
+    self->super.ops.create_rsc     = uct_cuda_copy_ctx_rsc_create;
+    self->super.ops.destroy_rsc    = uct_cuda_copy_ctx_rsc_destroy;
+    self->super.ops.progress_queue = uct_cuda_copy_progress_event_queue;
```

Comment:
IMO ops could point to a global struct, no need to save all pointers directly on the iface